### DISCUSSION TITLE
原神5.0前瞻兑换码

### DISCUSSION THREAD

**Discussion by yingguan-Cloud** [share121/inter-knot#2385] [其他]
5WJNV92GGUjW
9FJTU83HYDJA
XX3PD82ZYDGN

兑换码将于2024/8/16 12:00过期，请各位旅行者兼绳匠尽快兑换！
<img src="https://img-static.mihoyo.com/communityweb/upload/726f998f6a9c9a6979c2d199b5aa8d90.png" alt="是大佬" width="75" >

**Comment by nhb-bcd** [comment]
6

**Comment by 13346254991** [comment]
byd 过期了才发出来是吧
